README.rst

```diff
@@ -174,18 +174,17 @@ DataChain can parallelize API calls; the free Mistral tier supports up to 4 requ
 
 .. code:: py
```

Comment:
Please replace `pip install mistralai` to something like `pip install mistralai>=1.0.0` (please double check the exact command).

Reply:
> Please replace `pip install mistralai` to something like `pip install mistralai>=1.0.0` (please double check the exact command).

Ok,I will update it.